eureka/S5_lightcurve_fitting/s5_fit.py

```diff
@@ -161,25 +171,40 @@ def fitJWST(eventlabel, s4_meta=None):
                                 else:
                                     new_file.write(line)
 
-                # Set the intial fitting parameters
-                params = p.Parameters(param_file=meta.fit_par)
-
                 # Subtract off the zeroth time value to avoid floating point precision problems when fitting for t0
                 t_offset = np.floor(meta.bjdtdb[0])
                 t_mjdtdb = meta.bjdtdb - t_offset
                 params.t0.value -= t_offset
 
-                # Get the flux and error measurements for the current channel
-                flux = meta.lcdata[channel,:]
-                flux_err = meta.lcerr[channel,:]
-
-                # Normalize flux and uncertainties to avoid large flux values
-                flux_err = flux_err/ flux.mean()
-                flux = flux / flux.mean()
+                # Temporary normalization to avoid large flux values (FINDME: replace when constant offset is implemented)
+                flux = meta.lcdata[0,:] / np.mean(meta.lcdata[0,:200])
+                flux_err = meta.lcerr[0,:] / np.mean(meta.lcdata[0,:200])
+                for i in np.arange(meta.nspecchan-1):
+                    flux = np.concatenate((flux,meta.lcdata[i+1,:] / np.mean(meta.lcdata[i+1,:200])))
+                    flux_err = np.concatenate((flux_err,meta.lcerr[i+1,:] / np.mean(meta.lcdata[i+1,:200])))
 
                 # Load the relevant values into the LightCurve model object
-                lc_model = lc.LightCurve(t_mjdtdb, flux, channel, meta.nspecchan, unc=flux_err, name=eventlabel)
-
+                lc_model = lc.LightCurve(t_mjdtdb, flux, 0, meta.nspecchan, unc=flux_err, name=eventlabel,share=True)
+                
+                #Make a long list of parameters
+                longparamlist=[ [] for i in range(meta.nspecchan)]
```

Comment:
If there are no shared parameters, doesn't this just simplify down to the same stuff as what is at the bottom of the fitters.py file? That should make combining the shared and unshared codes easier

Reply:
This code should be in the fitters.py file rather than here though in my opinion